procrastinate/connector.py

```diff
@@ -6,10 +6,16 @@
 QUEUEING_LOCK_CONSTRAINT = "procrastinate_jobs_queueing_lock_idx"
 
 
+Pool = Any
```

Comment:
I think this deserves a comment.

Reply:
Following the previous comment, if we cannot specify the pool when entering context, this line is not needed anymore.